compiler/runtime/Helpers.inc

```diff
@@ -315,7 +315,6 @@ SETVAL(TR_PPCarrayOr,TR_FSRH+61)
 SETVAL(TR_PPCarrayAnd,TR_FSRH+62)
 SETVAL(TR_PPCarrayCmp,TR_FSRH+63)
 SETVAL(TR_PPCoverlapArrayCopy,TR_FSRH+64)
-SETVAL(TR_PPCarrayCmpVMX,TR_FSRH+66)
```

Comment:
When removing definitions from this table, we usually renumber all the later entries to avoid leaving any gaps (although it looks like an earlier removal snuck through at some point that didn't do this). Please update the numbering to close this gap.

Reply:
Renumbered later enum entries to close the gap in the runtime helper table.